pkg/agent/util/iptables/interfaces.go

```diff
@@ -0,0 +1,43 @@
+//go:build !windows
+// +build !windows
+
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iptables
+
+import (
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"antrea.io/antrea/pkg/apis/controlplane/v1beta2"
+)
+
```

Comment:
I saw there are "type Interface interface" in the iptables.go, maybe it can also be moved here?

Reply:
Agree